### DISCUSSION TITLE
Thunderbird no longer synching properly with Google calendar for about a week

### DISCUSSION THREAD

**Discussion by peteg99** [kewisch/gdata-provider#423] [Support]
About a week ago I noticed that most of my calendar events are not displayed on the Thunderbird Calendar tab.
Looking at my Google calendar on Google shows about 150 appointments during March
Looking at my Google calendar on Thunderbird only shows 2 appointments for March

I use Provider for Google calendar 91.0.4
Thunderbird is release 91.7.0
I have tried disabling Provider and re-enabling it - makes no difference.

Any suggestions please
Thanks

**Comment by kyleamadio** [comment]
I have a similar issue -   No writable calendars are configured for invitations, please check the calendar properties.

If I save the calendar item via GMAIL it will synch - however it does not work within Thunderbird - I have TBird 91.9.0

**Reply by ghost** [reply]
I am on TB v102.0.2.  I am also having the "No writable calendars..." issue.  Other aspects of google calendar appear to be working but not all.  Bidirectional sync works.  An event created in TB will sync to google - but not completely.  Invitees in the calendar entry are NOT being notified and the invitees are not replicating to Google.  Adding one person to the invite generates this warning twice

 - [calGoogleCalendar] 2022-07-18T21:00:00-04:00 does not match timezone offset for America/New_York - gdatautils.jsm:275

AND the invited person is stripped out of (or never saved to) the local calendar entry.  I'm just going between two of my own email accounts for testing.  Willing to help if I can.   TB #v102.0.2   Provider #v102.0.0

**Comment by wsmwk** [comment]
Are you still seeing this problem when using a newer version of Thunderbird and a newer version of provider?

**Reply by ghost** [reply]
FWIW it seems the solution for me (on TB v102.x.x on Windows 10) was to just remove the existing calendar(s) and reconnect them.  Using the new version of TB, you do not need the provider or other extensions to have rw access to your google calendar.  I tried for a week or so to make it work.  Everything has been working fine since.  I also run TB v91.11 on Linux and this has not been a problem.

**Comment by peteg99** [comment]
Yes - the newer version of Provider has made it all work again
Appreciate the help offered here :)

**Reply by davidselig** [reply]
After working well , the Calendar is again "bugging" either extremely slow to appear in TB, or not at all.  Synching with Google takes... too long!  Solutions?   Thanks for any help!

**Comment by nonilevinson** [comment]
I'm having the same problem on TB 102.2.2 (64-bit).
Yesterday I started using Provider for Google Calendar (v 102.0.0) again, but I don't like the with Provider I can not edit an event in one click.

**Reply by jpyllman** [reply]
For me time and time again I get a lot of synchronizing stuck in the "Activity Manager". So I kind of think my problem is the throttling. After putting all my calendars to manual sync and then starting Thunderbird I kind of got the calendar back working. So now have to wait and see how long it stays OK. And if I can turn 60m sync on the important calendars.

**Reply by nonilevinson** [reply]
Yesterday my sync return to work as we expect. My sync is to 15m at the main one and 60m on other three. I disabled the same four that I have with Provider for Google Calendar
